### PR TITLE
Updated download

### PR DESCRIPTION
**Pull request by finn00dev**

### CODE REVIEW

File: www/html/ci/application/views/landing.php [hunk 1]
```diff
@@ -669,9 +669,9 @@ function googleTranslateElementInit() {
 <p>Combine the cuteness of mochi sticky rice balls and a classic game of speed hockey. You get Dango Puck! With the exception of the music, everything was made by me. All graphics, code, and audio clips were made by me.</p>
 
 <p><strong>Download the game for </strong>
-<a href="http://www.mediafire.com/file/ty75vbtmqx5450m/DangoPuckv1MacUniversal.zip" target="_blank" >MacOS</a>, 
-<a href="http://www.mediafire.com/file/yl1w0b261212heb/DangoPuckv1Win86.zip" target="_blank" >Windows</a>, or
-<a href="http://www.mediafire.com/file/hpelalbkzni12m2/DangoPuckv1LinuxUniversal.zip" target="_blank" >Linux Universal.</a>
+<a href="<?php echo base_url(); ?>downloads/DangoPuckv1MacUniversal.zip" download="DangoPuckv1MacUniversal" >MacOS</a>,
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: @finn00dev I don't want to leave out the file extension type. Personally, I expect to see the .png or .jpeg when i see image files on my computer file system.
Just remove the renaming: <...download>

**Comment by ktorii** [comment]
@finn00dev You have a conflict. Pull the original repo's develop branch contents and merge it into your local branch. Fix the conflict and push the changes.

**Comment by finn00dev** [comment]
Ok I fixed the conflict

File: www/html/ci/application/views/landing.php [hunk 2]
```diff
@@ -519,7 +519,7 @@ function googleTranslateElementInit() {
 		<div class="portfolio-hover-content">
 		</div>
 		</div>
-		<a class="portfolio-link" href="https://drive.google.com/file/d/0B7TI-OA8_nbqb09SaTRlb3Y2TUE/view?usp=sharing" target="_blank"><img alt="" class="img-responsive" src="<?php echo base_url(); ?>img/derivativeskenvideo.jpg" /> </a>
+		<a class="portfolio-link" href="https://drive.google.com/file/d/0B7TI-OA8_nbqb09SaTRlb3Y2TUE/view?usp=sharing" target="_blank"><img alt="" class="img-responsive" src="<?php // echo base_url(); ?>img/derivativeskenvideo.jpg" /> </a>
```
Comment (ktorii) [inline www/html/ci/application/views/landing.php]: What is this

Comment (ktorii) [reply www/html/ci/application/views/landing.php]: Oh nevermind. You fixed this already.

**Comment by finn00dev** [comment]
Ok, the file extensions remain now

**Pull request merged**